### DISCUSSION TITLE
Small Error in documentation

### DISCUSSION THREAD

**Discussion by siedan17** [mathLab/EZyRB#241] [General]
Hi!

I looked through the source code of the reducedorder model. 
In there I found the description of the crossvalidation function. It said that if n_splits = num_of_snapshots, then it is the same as loo_error() except that here the relative error will be computed instead of the absolute (loo_error). 

From my inspection, I think that both functions compute the relative error. Am I wrong?

Kind regards,
Daniel

**Comment by ndem0** [comment]
Yes, you are right, it derives from one of the previous versions!
Thanks for spotting it! I'll correct asap
